ddtrace/internal/coverage/instrumentation.py

```diff
@@ -14,6 +15,9 @@ def instrument_all_lines(code: CodeType, hook: HookType, path: str) -> t.Tuple[C
 
     last_lineno = None
     for i, instr in enumerate(abstract_code):
+        if isinstance(instr, bytecode_instr.Label):
```

Comment:
```suggestion
        if isinstance(instr, Label):
```

Reply:
🤔 Is this check really needed? I'd have thought that a label would cause an exception within the `try` block below.

Reply:
It does. I just thought it was better, for code readability, to explicitly ignore types we know we want to ignore, rather than let the exception happen (although in terms of code readability, there was already the comment there 😓 ).

But maybe the `isinstance()` check's more costly, in this case, and since speed is of the essence here, I'm actually introducing a regression?

Reply:
FWIW, it looks like `isinstance()` is faster:
```
% python -m timeit -n 10000 -s "from bytecode import Label; labels = [Label() for _ in range(1000)]" "
for label in labels:
    if isinstance(label, Label):
        continue
    try:
        if label.lineno is None:
            continue
    except AttributeError:
        pass
"
10000 loops, best of 5: 12.1 usec per loop
```
vs
```
% python -m timeit -n 10000 -s "from bytecode import Label; labels = [Label() for _ in range(1000)]" "
for label in labels:
    try:
        if label.lineno is None:
            continue
    except AttributeError:
        pass
"
10000 loops, best of 5: 243 usec per loop
```